package.json

```diff
@@ -42,7 +42,7 @@
     "@stylistic/eslint-plugin": "^5.2.2",
     "@vitest/coverage-v8": "^3.2.4",
     "@vitest/eslint-plugin": "^1.3.4",
-    "dependency-cruiser": "^16.10.4",
+    "dependency-cruiser": "^17.0.0",
```

Comment:
This dropped support for older versions of Node: https://github.com/sverweij/dependency-cruiser/releases/tag/v17.0.0

Reply:
No. Odd-numbered releases aren't [LTS](https://nodejs.org/en/about/previous-releases); we'll never run them in development or production.